apps/functions-management/package.json

```diff
@@ -7,12 +7,15 @@
     "devDependencies": {
         "@pagopa/openapi-codegen-ts": "^13.1.0",
         "@tsconfig/node20": "^20.1.4",
+        "@types/jest": "^29.5.12",
         "azure-functions-core-tools": "^4.0.5700",
         "eslint": "^9.2.0",
         "eslint-config-rules": "workspace:^",
+        "jest": "^29.7.0",
```

Comment:
consider that jest is on hold in our technology radar https://pagopa.github.io/technology-radar/tools/jest.html

Reply:
Moreover, in the PR that init the structure of the first function [we already have initialized `vitest`](https://github.com/pagopa/trial-system/blob/features/init-functions-subscription/vitest.config.ts)